src/tests/benchmark/atomdb/atomdb_utils.h

```diff
@@ -58,5 +59,6 @@ double measure_execution_time(Func&& func) {
     auto start = chrono::steady_clock::now();
     func();
     auto end = chrono::steady_clock::now();
-    return chrono::duration<double, milli>(end - start).count();
+    double duration_millis = chrono::duration<double, milli>(end - start).count();
+    return round(duration_millis * 100.0) / 100.0;  // Round to 2 decimal places
```

Comment:
You shouldn't cut precision when you are measuring or manipulating numbers in general. You only do such simplification when you are about to report to a text file or a GUI component. Internally, you ALWAYS keep maximum precision. If cut precision too early, like you did here, the rounding error will sum up whenever you use this number downstream.

When we integrate to the DB. You put the whole precision in the DB as well. Let whoever is showing these numbers to the final caller to cut down precision, if necessary.